draft-dejong-remotestorage-06.txt

```diff
@@ -374,15 +384,15 @@ Internet-Draft              remoteStorage                  November 2015
     already exists.
 
     In all 'ETag', 'If-Match' and 'If-None-Match' headers, revision
-    strings should appear inside double quotes (").
+    strings, except '*', should appear inside double quotes (").
```

Comment:
We're describing HTTP syntax here. This is already defined elsewhere. Is this really needed?